pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -66,9 +66,9 @@ type ResolvedPipelineTask struct {
 	// If the PipelineTask is a Custom Task, RunName and Run will be set.
 	CustomTask            bool
 	RunName               string
```

Comment:
Renaming this to `RunObjectName` will make it clearer that it's either Run or CustomRun name